.travis.yml

```diff
@@ -2,7 +2,12 @@ sudo: false
 dist: trusty
 language: python
 python:
-  - 2.7
-services: rabbitmq
-install: pip install -r requirements.txt
-script: cd src; TXAMQP_BROKER=RABBITMQ trial txamqp
+  - "2.7"
+  - "3.3"
+  - "3.4"
```

Comment:
FWIW I don't think we need 3.3 and 3.4. I believe they are becoming obsolete these days and have very few users. Other projects are dropping support for 3.3 and 3.4 too.